packages/react-composites/src/composites/MeetingComposite/adapter/parseTeamsUrl.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * Get chat thread from a teams url.
+ * As per documented on the Chat SDK: https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/chat/meeting-interop?pivots=platform-web#get-a-teams-meeting-chat-thread-for-a-communication-services-user
+ *
+ * @private
+ */
+export const getChatThreadFromTeamsLink = (teamsMeetingLink: string): string => {
+  // Get the threadId from the url - this also contains the call locator ID that will be removed in the threadId.split
+  let threadId = teamsMeetingLink.replace('https://teams.microsoft.com/l/meetup-join/', '');
```

Comment:
Are we certain this will never change?

Reply:
As sure as we can be unfortunately. I'm trying to get the chat sdk to support taking in a teams url. If they do then we will remove this manual extraction (though then the same problem shifts to them).

I discussed this with the calling team (as they do support taking the entire url) and they said they were just OK with taking the risk if it does change and have a relationship with teams to try make sure it doesn't